### ISSUE TITLE
Consider adding tags for each release

### ISSUE THREAD

**Issue by ftomassetti** [LionWeb-io/lionweb-typescript#149]
For debugging, it could be a nice to have to be able to connect each version released on NPM with a specific tag in the repository.
Perhaps using [changeset](https://github.com/changesets/changesets), as the lionweb-repository started doing could help.

**Comment by dslmeinte** [comment]
I thought I did that (manually), but it turns out I didn't. I'll add tags after-the-fact now. Using `changeset` is certainly a good idea – I'll consider that the second part of this issue, maybe to-be-done after we've gained some experience on `lionweb-repository`.

**Comment by dslmeinte** [comment]
Also, @ftomassetti : I think it's inevitable to have one tag per package (`core`, `utilities`, `validation`, `cli`) per version – agree?

**Comment by joswarmer** [comment]
As we publish packages separately, we need a tag per package indeed. This is different from the `lionweb-repository`.

**Comment by dslmeinte** [comment]
Yes, I would indeed use the format `<package>-<version>`. The reason for the “asynchronicity” is a fix of the `cli` package that only afflicted that package and none of the others. As soon as we hit `0.7.0` I'll align all the versions again.
